mountstats.go

```diff
@@ -534,17 +536,33 @@ func parseNFSOperationStats(s *bufio.Scanner) ([]NFSOperationStats, error) {
 
 			ns = append(ns, n)
 		}
-
-		opStats := NFSOperationStats{
-			Operation:                           strings.TrimSuffix(ss[0], ":"),
-			Requests:                            ns[0],
-			Transmissions:                       ns[1],
-			MajorTimeouts:                       ns[2],
-			BytesSent:                           ns[3],
-			BytesReceived:                       ns[4],
-			CumulativeQueueMilliseconds:         ns[5],
-			CumulativeTotalResponseMilliseconds: ns[6],
-			CumulativeTotalRequestMilliseconds:  ns[7],
+		var opStats NFSOperationStats
+		if ns[0] != 0 {
```

Comment:
Instead of this assign opStats without AverageRTT, then check if ns[0] != 0 and add AverageRTT

Reply:
thanks for review. I made the changes.